src/index.css

```diff
@@ -14,6 +14,14 @@
 	--color-border: hsla(220, 13%, 32%, 1);
 	--color-error: var(--color-red);
 	--color-text: var(--color-white);
+
+	--color-branding: #5a23b3;
+	--btn-bg-color-confirm: var(--color-branding);
+	--btn-bg-color-confirm-hover: #3f0f7f;
+	--btn-bg-color-cancel: #cdcdcd;
+	--btn-bg-color-cancel-hover: #989797;
+	--btn-bg-color-delete: var(--color-error);
+	--btn-bg-color-delete-hover: #a80000;
```

Comment:
🔍 I like this, but I think we could bring _even more clarity_, theoretically. These variable names describe the function of the colors, but don't tell someone at a glance what the colors are. Is `--btn-bg-color-confirm-hover` green? red? dandelion-ish?

In my time, I've seen CSS vars set up in 2 stages, with the first stage being the one that gives human-readable names to the colors, and the second stage assigning the function of the colors. I did something like this initially, although I didn't do the best job:

```css
--color-red: hsl(0, 68%, 42%);
```

We know that this variable is a red, as opposed to a blue or a green. I think it makes sense to use _that_ name when picking the background color for your button:

```css
--btn-bg-color-delete: var(--color-red);
```

All of your other colors could follow the same pattern: first name the colors, then attach them to the elements they're used in.

Reply:
Ah ok I understand that now. That way when someone else looks at the line defining the --btn variable they will see a color named versus a hsl/hex/rgb. And I guess this also allows for that color to be reusable throughout the app as well?